### PR TITLE
Completed data structures stacks exercise - Daniel Cho

### PR DESCRIPTION
**Pull request by Daniel-J-Cho**
Screen Shot 2022-06-13 at 11 49 11 PM
Screen Shot 2022-06-13 at 11 49 23 PM

### CODE REVIEW

File: data-structures-stacks/get-2nd-from-top.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* exported get2ndFromTop */
+
+function get2ndFromTop(stack) {
+  if (stack.peek() === undefined) {
+    return undefined;
+  }
+  const poppedVal = stack.pop();
+  if (stack.peek() === undefined) {
+    stack.push(poppedVal);
+  } else if (stack.peek() !== undefined) {
```
Comment (scbowler) [inline data-structures-stacks/get-2nd-from-top.js:10]: Just an `else` here would be better, there is no need to check the exact opposite of what you already checked, that's what the `else` is for.

File: data-structures-stacks/max-value.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/* exported maxValue */
+
+function maxValue(stack) {
+  if (stack.peek() === undefined) {
+    return -Infinity;
+  } else if (stack.peek() !== undefined) {
```
Comment (scbowler) [inline data-structures-stacks/max-value.js:6]: Same here just an `else` would be better

**Review by scbowler — APPROVED** [review]
Looks good! 👍 

A couple FYI comments, no changes required.

File: data-structures-stacks/get-2nd-from-top.js (hunk 1 above)
Comment (Daniel-J-Cho) [reply data-structures-stacks/get-2nd-from-top.js:10]: Thank you Scott.

File: data-structures-stacks/max-value.js (hunk 2 above)
Comment (Daniel-J-Cho) [reply data-structures-stacks/max-value.js:6]: I seem to have a bad habit here.  Thanks again Scott

**Pull request merged**